metals/src/main/scala/scala/meta/metals/DownloadDependencies.scala

```diff
@@ -30,17 +42,79 @@ object DownloadDependencies {
    * @param args ignored.
    */
   def main(args: Array[String]): Unit = {
+
+    val filterVersions: String => Boolean =
+      args.indexOf("--scala-versions") match {
+        case -1 => (_: String) => true
+        case index =>
+          val versions =
+            args.drop(index + 1).takeWhile(!_.startsWith("-")).toSet
+          println("Filtering versions: " + versions.mkString(", "))
+          (s: String) => versions(s)
+      }
     MetalsLogger.updateDefaultFormat()
-    val allPaths = downloadMdoc() ++
+    val allPaths = downloadMdoc(filterVersions) ++
       downloadScalafmt() ++
-      downloadMtags() ++
-      downloadSemanticDBScalac() ++
+      downloadMtags(filterVersions) ++
+      downloadSemanticDBScalac(filterVersions) ++
       downloadSemanticDBJavac() ++
       downloadScala() ++
       downloadBloop() ++
-      downloadBazelBsp()
+      downloadBazelBsp() ++
+      downloadScala3PresentationCompiler(filterVersions) ++
+      downloadAllScalafixVersions(filterVersions)
+
+    val distinctFiles = allPaths.distinct
+    val copytDest = args.indexOf("--copy-to") match {
+      case -1 => None
+      case index =>
+        Some(args(index + 1))
+    }
+
+    copytDest.foreach { path =>
+      copyToDestination(distinctFiles, path)
+    }
+    distinctFiles.foreach(println)
```

Comment:
Should we make this println more descriptive?

Reply:
That's on purpose so that people don't have to parse the output if they want to use it